.github/workflows/build.yml

```diff
@@ -0,0 +1,116 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+      - gh-actions # TODO: remove this.
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+  schedule:
+    - cron: '0 4 * * *' # Runs every day at 4am: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule
+
+jobs:
+
+  build:
+    name: ${{ matrix.os }}, Node.js v${{ matrix.node }}, Python ${{ matrix.python }}
+
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
```

Comment:
I think we should not use aliases there but use real versions
like macos-10.15, ubuntu-18.04 or ubuntu-20.04 as ubuntu-latest is switching to 20.04 soon on github actions

it will prevent to have scripts that are no longer working just due to a switch of runners that are not under control

Reply:
> like macos-10.15, ubuntu-18.04 or ubuntu-20.04 as ubuntu-latest is switching to 20.04 soon on github actions

Sure, no problem. Let me know which targets do you prefer. Thanks!

Reply:
just replace latest by their current counter part specified now as you already tested these configurations
https://github.com/actions/virtual-environments#available-environments

it's then: `ubuntu-18.04` `macos-10.15` `windows-2019`